thanos/README.md.gotmpl

```diff
@@ -257,9 +296,9 @@ To enable the creation of a ServiceMonitor we can use the Thanos Plugin configur
 ```yaml
 spec:
   optionsValues:
-  - name: thanos.ruler.serviceMonitor.selfMonitor
+  - name: thanos.serviceMonitor.selfMonitor
       value: true
-  - name: thanos.ruler.serviceMonitor.additionalLabels
+  - name: thanos.serviceMonitor.labels
```

Comment:
We don't need additional labels anymore?

Reply:
aligning docs to values.yaml
https://github.com/cloudoperators/greenhouse-extensions/blob/main/thanos/charts/values.yaml#L34